clinica/pipelines/dwi_preprocessing_using_t1/dwi_preprocessing_using_t1_workflows.py

```diff
@@ -82,7 +82,7 @@ def eddy_fsl_pipeline(low_bval, use_cuda, initrand, name="eddy_fsl"):
     return wf
 
 
-def epi_pipeline(name="susceptibility_distortion_correction_using_t1"):
+def epi_pipeline(self, name="susceptibility_distortion_correction_using_t1"):
```

Comment:
You should pass the arguments you actually need here, which seems to be:

- a boolean to know whether you should clean the cache or not, defaulting to "not clean the cache"
- the `base_dir` path if there is no other way to recover it

Please also update the docstring to describe these new parameters.